### PR TITLE
make robust to bad sortingPriority values

### LINKED ISSUE #755: Make robust to bad sortingPriority values
having bad sortingPriority values in airtable severely messes up the app in baffling and terrifying ways. Fix the code so that it treats bad values as "low".

### LINKED ISSUE #756: check for bad sortingPriority values
add a test that all benefits have a `sortingPriority` value that is empty, low, medium, or high.

### PR DESCRIPTION
**Pull request by sastels**
fixes #755

### CODE REVIEW

File: __tests__/components/benefit_list_test.js [hunk 1]
```diff
@@ -44,6 +44,17 @@ describe("BenefitList", () => {
...
   });
 
+  it("has a correct cleanSortingPriority function", () => {
+    let BLInstance = shallow(
+      <BenefitList {...props} {...reduxData} />
+    ).instance();
+    expect(BLInstance.cleanSortingPriority("high")).toEqual("high");
+    expect(BLInstance.cleanSortingPriority("medium")).toEqual("medium");
+    expect(BLInstance.cleanSortingPriority("low")).toEqual("low");
+    expect(BLInstance.cleanSortingPriority("bad value")).toEqual("low");
```
Comment (smcmurtry) [inline __tests__/components/benefit_list_test.js:54]: Nice!

**Review by smcmurtry — APPROVED** [review]
Looks good! 🦑
